### ISSUE TITLE
Overwrite existing assets 

### ISSUE THREAD

**Issue by AtiqGauri** [shogo82148/actions-upload-release-asset#66]
Does it support overwriting existing assets...    
OR there is any way we can do that?

**Comment by shogo82148** [comment]
No, the action doesn't support overwriting.
You have to delete the assets using [Delete a release asset](https://developer.github.com/v3/repos/releases/#delete-a-release-asset) API before uploading.

I recommend you to create a new release and upload assets.

**Comment by AtiqGauri** [comment]
It would be great if you can add this as an optional argument... 

And can you help me to give an example to delete that asset

**Comment by shogo82148** [comment]
OK. I'll try it later.

The integrated test may help you.
https://github.com/shogo82148/actions-upload-release-asset/blob/361a849ab8b54c0f3c166b0a02d3357809a8f63a/.github/workflows/test.yml#L56-L58

**Comment by shogo82148** [comment]
🤔 
It seems that we can't get the repo name from `${{ steps.create_release.outputs.upload_url }}` if uploading fails.

```
$ curl -H "Accept: application/vnd.github.v3+json" -H "Authorization: token $GITHUB_TOKEN" 'https://uploads.github.com/repos/shogo82148/github-action-test/releases/23245222/assets?name=release.yml' -X POST -H 'Content-Type: text/plain' -d 'Hello GitHub' -v
(snip)
< HTTP/1.1 422 Unprocessable Entity
< Date: Thu, 16 Jul 2020 23:48:02 GMT
< Content-Type: application/json; charset=utf-8
< Content-Length: 209
< Cache-Control: no-cache
< Content-Security-Policy: default-src 'none'
< Strict-Transport-Security: max-age=31557600
< Vary: Accept-Encoding, Accept, X-Requested-With
< X-Accepted-Oauth-Scopes: repo
< X-Content-Type-Options: nosniff
< X-Frame-Options: deny
< X-Github-Media-Type: github.v3; format=json
< X-Oauth-Scopes: read:packages, repo, workflow, write:packages
< X-Xss-Protection: 1; mode=block
< X-GitHub-Request-Id: C2F2:5B59:E9BF4:FC623:5F10E731
<
* Connection #0 to host uploads.github.com left intact
{"message":"Validation Failed","request_id":"C2F2:5B59:E9BF4:FC623:5F10E731","documentation_url":"https://developer.github.com/v3","errors":[{"resource":"ReleaseAsset","code":"already_exists","field":"name"}]}
```

**Comment by shogo82148** [comment]
> GitHub renames asset filenames that have special characters, non-alphanumeric characters, and leading or trailing periods. The "List assets for a release" endpoint lists the renamed filenames. For more information and help, contact GitHub Support.

We need to know this rename rule 🤔

**Comment by AtiqGauri** [comment]
Checkout [this action](https://github.com/marketplace/actions/upload-files-to-a-github-release), it has a overwrite optional argument.
My problem with that action is, it doen't upload to a upload_url instead it create a new release with tag_name

**Comment by shogo82148** [comment]
I just released [v1.2.0](https://github.com/shogo82148/actions-upload-release-asset/releases/tag/v1.2.0) that supports `overwrite` flag.
please try it!
